app/components/provider_interface/activity_log_event_component.rb

```diff
@@ -0,0 +1,98 @@
+module ProviderInterface
+  class ActivityLogEventComponent < ViewComponent::Base
+    include ViewHelper
+    attr_reader :event, :application_choice
+
+    ORIGINAL_OPTION_STATUSES = %w[awaiting_provider_decision rejected].freeze
+
+    def initialize(activity_log_event:)
+      @event = activity_log_event
+      @application_choice = event.auditable
```

Comment:
Maybe consider using `activity_log_event` so it's more obvious that `auditable` is retrieved from the method parameter - I had to look around to figure out that event is the same object.